docs/plugins.md

```diff
@@ -13,10 +13,10 @@ Let's take a quick look at the current dokku nginx plugin that's shipped with do
 ## Installing a plugin
 
 ```shell
-cd /var/lib/dokku/plugins/available
-git clone <git url>
-dokku plugin:install
+dokku plugin:install <git url>
 ```
+Note: `plugin:install` might trigger actions like `apt-get`, please make sure you inspect any potential plugin's `install` or `install-dependencies` trigger, and (of course) execute this command as the `root` user
+
```

Comment:
`/var/lib/dokku` is owned by `root`. However we don't install plugins in that directory anymore.